metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/LogConstants.java

```diff
@@ -13,95 +13,109 @@
 
 package com.netflix.metacat.common.server.monitoring;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+//CHECKSTYLE:OFF
 /**
  * Log constants.
  */
 public enum LogConstants {
```

Comment:
It can be defined as a constant class. But it also has methods, and needs to be a singleton. Enum seems ok. The naming is bit misleading, and should be renamed as Metrics